src/studio/src/designer/frontend/dashboard/App.tsx

```diff
@@ -90,10 +99,25 @@ export const App = () => {
             exact={true}
             component={StandaloneDataModelling}
           />
-        </div>
+        </div> : <Grid justifyContent='center'>
+          <AltinnSpinner spinnerText='Venter på svar' />
+          {showLogOutButton && <AltinnButton
+            onClickFunction={handleLogout}
+            btnText={'Logg ut'}
+          />}
+        </Grid>
+        }
       </Router>
     </MuiThemeProvider>
   );
 };
 
 export default App;
+const handleLogout = () => {
+  const altinnWindow: Window = window;
```

Comment:
This can be simplified. No need to create a separate `altinnWindow` variable for this, and the return statement is redundant.

```javascript
const handleLogout = () => {
  const url = `${window.location.origin}/repos/user/logout`;
  post(url).then(() => {
    window.location.assign(`${window.location.origin}/Home/Logout`);
  });
}
```

Reply:
aggree, I copied it from one of the other duplicates to get it done in e jiff. I'll clean that up.

- [ ] clean up handleLogout